lms/envs/common.py

```diff
@@ -2215,4 +2215,4 @@
 ECOMMERCE_API_TIMEOUT = 5
 
 # Reverification checkpoint name pattern
-CHECKPOINT_PATTERN = r'(?P<checkpoint_name>\w+)'
+CHECKPOINT_PATTERN = r'(?P<checkpoint_name>[a-zA-Z0-9_ ]+)'
```

Comment:
The `\w` sequence matches alphanumeric characters (`[0-9a-zA-Z_]`). That being the case, why can't you continue using `\w+` here? Is matching on the whitespace character intentional? If so, I'd suggest using `[\w ]+`.